<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VeggGUI</class>
 <widget class="QWidget" name="VeggGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VeggGUI</string>
  </property>
  <widget class="QListWidget" name="veggieList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="restList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLine">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="resultList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
